src/common/vitals/interaction-to-next-paint.js

```diff
@@ -7,7 +7,14 @@ export const interactionToNextPaint = new VitalMetric(VITAL_NAMES.INTERACTION_TO
 
 if (isBrowserScope) {
 /* Interaction-to-Next-Paint */
-  onINP(({ value, entries, id }) => {
-    interactionToNextPaint.update({ value, entries, attrs: { metricId: id } })
```

Comment:
This is changing the signature of the object we report out via the update method. Where does this change need to be accounted for and has it been?